actor/v7action/service_app_binding.go

```diff
@@ -54,6 +60,48 @@ func (actor Actor) CreateServiceAppBinding(params CreateServiceAppBindingParams)
 	}
 }
 
+func (actor Actor) DeleteServiceAppBinding(params DeleteServiceAppBindingParams) (chan PollJobEvent, Warnings, error) {
+	var (
+		serviceInstance resources.ServiceInstance
+		app             resources.Application
+		binding         resources.ServiceCredentialBinding
+		jobURL          ccv3.JobURL
+		stream          chan PollJobEvent
+	)
+
+	warnings, err := railway.Sequentially(
+		func() (warnings ccv3.Warnings, err error) {
+			serviceInstance, _, warnings, err = actor.getServiceInstanceByNameAndSpace(params.ServiceInstanceName, params.SpaceGUID)
+			return
```

Comment:
Just out of curiosity, are those `return` statements needed? 🤔

Reply:
Yes, you always need a `return` in Go if the function has a return value. With named return values you can omit the variable names, but you still need the `return` statement.

Reply:
Yeah It just passed me that the function signature has return values 🤦